### ISSUE TITLE
istioctl analyze doesn't understand operator annotations

### ISSUE THREAD

**Issue by howardjohn** [istio/istio#18621]
```
Warn [IST0108] (Service jaeger-collector.istio-system) Unknown annotation: install.operator.istio.io/owner-generation
Warn [IST0108] (Service jaeger-query.istio-system) Unknown annotation: install.operator.istio.io/chart-owner
Warn [IST0108] (Service jaeger-query.istio-system) Unknown annotation: install.operator.istio.io/owner-generation
Warn [IST0108] (Service prometheus.istio-system) Unknown annotation: install.operator.istio.io/chart-owner
Warn [IST0108] (Service prometheus.istio-system) Unknown annotation: install.operator.istio.io/owner-generation
Warn [IST0108] (Service tracing.istio-system) Unknown annotation: install.operator.istio.io/chart-owner
Warn [IST0108] (Service tracing.istio-system) Unknown annotation: install.operator.istio.io/owner-generation
Warn [IST0108] (Service zipkin.istio-system) Unknown annotation: install.operator.istio.io/chart-owner
Warn [IST0108] (Service zipkin.istio-system) Unknown annotation: install.operator.istio.io/owner-generatio
```

Probably needs to be added to a list somewhere, I think there is one is istio.io/api?

**Comment by ayj** [comment]
@sushicw, should this be resolved once the operator annotations are added to istio/api?

**Comment by sushicw** [comment]
It looks like it's currently a hand-crafted list: https://github.com/istio/istio/blob/master/galley/pkg/config/analysis/analyzers/annotations/annotations.go#L32-L79

So it won't be automatic, but once the operator annotations are in istio/api we can add them to the list here.

**Comment by sushicw** [comment]
@howardjohn @ayj Are these annotations going to be in 1.4?

**Comment by howardjohn** [comment]
Yes, we should cherrypick these to 1.4

Every resource created by istioctl/operator has these

**Comment by sushicw** [comment]
Keeping this open until this is merged into 1.4.
